bin/bundler/src/bundler.rs

```diff
@@ -98,9 +101,22 @@ fn main() -> Result<()> {
                     }
                 }
 
-                let wallet = Wallet::from_file(opt.mnemonic_file.clone(), &chain_id)
+                let wallet: Wallet;
+                if opt.fb_mnemonic_file.is_some() {
```

Comment:
Is there any specific reason for having different mnemonic phrase for flashbots?

Reply:
I was following [the guideline of Flashbots](https://docs.flashbots.net/flashbots-auction/searchers/quick-start#how-to-send-your-first-flashbots-bundle). In particular, it shouldn't store any funds

```
When you send bundles to Flashbots you will sign them with a private key 
so that we can establish identity for searchers and establish a reputation for them over time. 
This private key does not store funds and is not the primary private key you use for executing transactions. 
```

Reply:
Yeah, I get that. But we can use different derivation paths and single seed phrase (that ways bundler admin don't have to store two seed phrases):

 "m/44'/60'/0'/0/0" and  "m/44'/60'/0'/0/1". First for executing transactions and the second for flashbots identity.

`ethers-rs` allows changing derivation paths.

Reply:
Oh, nice. I didn't know that was possible. 

I'll make the change